plugin-server/tests/worker/vm/capabilities.test.ts

```diff
@@ -1,60 +1,148 @@
 import { Hub, LogLevel, PluginCapabilities } from '../../../src/types'
 import { createHub } from '../../../src/utils/db/hub'
-import { getVMPluginCapabilities } from '../../../src/worker/vm/capabilities'
+import { getVMPluginCapabilities, shouldSetupPluginInServer } from '../../../src/worker/vm/capabilities'
 import { createPluginConfigVM } from '../../../src/worker/vm/vm'
 import { pluginConfig39 } from '../../helpers/plugins'
 
-describe('getVMPluginCapabilities()', () => {
-    let hub: Hub
-    let closeHub: () => Promise<void>
+describe('capabilities', () => {
```

Comment:
Note: We don't really need this top-level describe.

Reply:
It's not _needed_ but it's how we do things since https://github.com/PostHog/posthog/pull/8955. If nothing else, it makes me running tests a better experience since I have these play buttons next to `describe`s